src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -288,11 +288,13 @@ protected function getModuleControllersFromRouting(string $moduleName): array
         }
 
         $routingControllers = [];
-        $moduleRoutes = $this->routingConfigLoader->import($routingFile, 'yaml');
-        foreach ($moduleRoutes->getIterator() as $route) {
-            $legacyController = $route->getDefault('_legacy_controller');
-            if (!empty($legacyController)) {
-                $routingControllers[] = $legacyController;
+        if (is_callable([$this->routingConfigLoader, 'import'])) {
```

Comment:
maybe `routingConfigLoader` should be a Loader instead of LoaderInterface :thinking:

Reply:
Oops. I forgot it. I fixed it and documented the BC ;)

Reply:
:heavy_check_mark: